opentok/opentok.py

```diff
@@ -453,7 +453,7 @@ def _create_jwt_auth_header(self):
                       'iss': self.api_key,
                       'iat': int(time.time()), # current time in unix time (seconds)
                       'exp': int(time.time()) + (60*3), # 3 minutes in the future (seconds)
-                      'jti': '{:f}'.format(random.random())
```

Comment:
https://docs.python.org/2/library/string.html

Changed in version 2.7: The positional argument specifiers can be omitted, so '{} {}' is equivalent to '{0} {1}'.

The previous version of interpolation used an advanced formatting only available in Python 2.7 - I updated it to use a less concise but compatible version with 2.6 and below.